rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -141,4 +141,22 @@ enum RocalDecoderType
 #define    ROCAL_MEMCPY_TO_DEVICE    2      // force copy to user provided device memory (gpu)
 #define    ROCAL_MEMCPY_IS_PINNED    4      // for future use
 
+enum RocalResizeScalingMode
+{
+    ROCAL_SCALING_MODE_DEFAULT = 0,
```

Comment:
Can we add short comments on what does these scaling modes mean? similar to 
https://github.com/shobana-mcw/MIVisionX/blob/90937c11605dd06ce05eb1a7f68a77416a134036/rocAL/rocAL/include/api/rocal_api_types.h#L108